lib/vm.js

```diff
@@ -46,7 +47,19 @@ class MetaScript {
 
 const createScript = (name, src, options) => new MetaScript(name, src, options);
 
+const metarequire = (context) => (module) => {
+  if (Reflect.has(context, module)) return Reflect.get(context, module);
+  try {
+    const src = fs.readFileSync(module, 'utf8');
```

Comment:
We have `readScript` in `lib/loader.js` so I think if we need it here it would be better to move everything from this module to a single file. I'll do that ASAP so you need to rebase this PR

Reply:
@AliusDieMorietur everything moved into a single file: https://github.com/metarhia/metavm/commit/cf62d8ffad3e30414507f62d6d4f283069049ead